docs/prerequisites.md

```diff
@@ -0,0 +1,48 @@
+# Prerequisites
+
+## Requirements
+
+- Linux or MacOS (Windows isn't supported at the moment).
+- A [Google Cloud](https://console.cloud.google.com) account.
+- [Packer](https://packer.io) to build images
```

Comment:
- Do you want to add `kustomize` (needed for the manifests?)
- also `ansible` is needed at least for now? (used from packer)

Reply:
+1 to ansible, kustomize should be optional at this point. Unless folks want to build the examples